python/tvm/relay/testing/tflite.py

```diff
@@ -0,0 +1,468 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+TensorFlow Lite model generation infrastructure that uses flatbuffers
+============================================================
+"""
+import json
+import subprocess
+import tempfile
+from enum import Enum
+from typing import List, Dict, Optional, Any, Tuple, Union
+import numpy as np
+from tvm.contrib.download import download
+
+# We are currently using TensorFlow Lite 2.4.2 schema to write the model buffers
+SCHEMA_URL = (
+    "https://raw.githubusercontent.com/tensorflow/tensorflow/v2.4.2/"
+    "tensorflow/lite/schema/schema.fbs"
+)
+
+
+class ActivationFunction(Enum):
+    NONE = "NONE"
+    RELU = "RELU"
+    RELU_N1_TO_1 = "RELU_N1_TO_1"
+    RELU6 = "RELU6"
+    TANH = "TANH"
+    SIGN_BIT = "SIGN_BIT"
+
+
+class Quantization:
+    "A class representing quantization of a tensor"
+
+    def __init__(
+        self,
+        scale: List[float],
+        zero_point: List[int],
```

Comment:
Given these are both lists, perhaps scales/zero_points?

Reply:
Yeah could do. I was thinking though that since we have multiple scales and zero points only when we have per channel quantization (i.e. only for the convolutions), it would not make much sense for all the other operators. I was also thinking to change it to accept both, a scalar and a list and convert the scalar to a list inside the class.